README.md

```diff
@@ -14,6 +14,10 @@ Two common problems observed in large Kubernetes clusters are:
 **node-reaper** provides the capability for worker nodes to be force terminated so that replacement ones come up.
 **pod-reaper** does a force termination of pods stuck in Terminating state for a certain amount of time.
 
+In some cases, on multi-tenant platforms, users can own PDBs which block node rotation/drain if they are misconfigured, pods are in crashloop backoff, or if multiple PDBs are targeting the same pods.
```

Comment:
Misbehaving worker nodes are okay to be terminated because they are brought back by various autoscaling mechanisms. Misbehaving pods are okay to be terminated because various higher level objects (Deployments, replicasets, etc.) bring back the terminated pods. Nodes and pods can be considered ephemeral in that sense.

However, this is not true with PDBs. If PDBs are deleted, users will have to find ways of bringing them back. It might be okay if they use GitOPS but in non-gitops envs, objects getting deleted could be problematic.

Not sure what's a good answer for preventing misbehaving PDBs from stopping node drains though. If there was a way to temporarily disable these PDBs during upgrades, that would've been ideal.

Reply:
Yes, I added a comment on the above in the individual readme
```
Since pdb-reaper will not recreate the PDBs it deletes, deletion is particularly useful in cases where GitOps is used, which can re-create the PDBs at a later time.
```

Reply:
Users who just want visibility can run with `--dry-run` and avoid deletion

Reply:
Maybe a future enhancement would be to save the PDB and then restore it later, presumably after the node was successfully drained? But then pdbreaper would need to be triggered when a node is about to be drained so the 'bad' PDBs can be temporarily deleted, like maybe triggered by upgrade-manager or lifecycle-manager. This is essentially what is recommended by [this GCP Anthos link](https://cloud.google.com/anthos/clusters/docs/bare-metal/1.7/troubleshooting/remove-node-issues).

Reply:
This adds a lot of complexity, you would have to track node drains and figure out the right time to recreate - this is easy when doing this manually (such as the doc you linked), but doing it programmatically is much more complex ( complexity which we might not want to have).